ctl/src/command.rs

```diff
@@ -591,10 +586,9 @@ pub fn metrics(mut channel: Channel<CommandRequest,CommandResponse>, json: bool)
           },
           CommandStatus::Error => {
             if json {
-              print_json_response(&message.message);
+              print_json_response(&message.message)?;
```

Comment:
Same here, you should return an error

Reply:
How about we do

```rust
print_json_response(&message.message)
```
with no `?` or `;` ?

Reply:
nevermind, I corrected as above